lib/web_ui/lib/src/engine.dart

```diff
@@ -15,10 +15,15 @@ library engine;
 // - https://github.com/flutter/engine/blob/master/web_sdk/sdk_rewriter.dart
 
 import 'dart:async';
+// ignore: unused_import
 import 'dart:collection'
     // Some of these names are used in services/buffers.dart for example.
     // ignore: unused_shown_name
-    show ListBase, IterableBase, DoubleLinkedQueue, DoubleLinkedQueueEntry;
```

Comment:
this will cause lint error due to //ignore not applied anymore. would be good to file a bug for dartfmt

Reply:
Ignoring the entire import now.